conf.d/server.conf.tmpl

```diff
@@ -243,4 +243,17 @@ server {
 
     ${HTTP_BASIC1}
   }
+
+  # For parse upload centre service
+  location /uploadcentre {
+    proxy_pass ${NGINX_PROXY_PARSE};
+
+    # Ensure HTTP get passed thru
+    proxy_pass_request_headers on;
+
+    # its helpful to cache these responses
+    proxy_cache globalcache;
```

Comment:
I am wondering if this needs to be here at all, since the URL is being used to upload files as far as my understanding goes... Is it helpful for the login headers maybe?